### PR TITLE
Clarify JSON types and their uses

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: e4c47111e6907c43ad180923bed9ac54b5f509cb
Build details: https://travis-ci.org/ampsauce/amphtml/builds/91637355

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by dvoytenko** [comment]
@ampsauce retry

File: src/json.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * @fileoverview This module declares JSON types as defined in the
+ * {@link http://json.org/}.
+ */
+
+
+/**
+ * JSON scalar. It's either string, number or boolean.
+ * @typedef {string|number|boolean}
+ */
+const JSONScalar = {};
```
Comment (cramforce) [inline src/json.js]: Probably better to make this
`let JSONScalar`;

**Comment by cramforce** [comment]
LGTM

File: src/json.js (hunk 1 above)
Comment (dvoytenko) [inline src/json.js]: Done

**Comment by ampsauce** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 6ab1a830d880e26ccbbb53dace2b29763cf2cfaa
Build details: https://travis-ci.org/ampsauce/amphtml/builds/91655617

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Pull request merged**